packages/blocks/src/specs/index.ts

```diff
@@ -7,8 +31,40 @@ SpecProvider.getInstance().addSpec('page', PageEditorBlockSpecs);
 SpecProvider.getInstance().addSpec('edgeless', EdgelessEditorBlockSpecs);
 SpecProvider.getInstance().addSpec('preview', PreviewEditorBlockSpecs);
 
+// specs preset
 export * from './edgeless-specs.js';
 export * from './page-specs.js';
 export * from './preview-specs.js';
 export * from './utils/spec-builder.js';
 export * from './utils/spec-provider.js';
+
```

Comment:
You can use

```ts
const commonSpec = {};
const edgelessSpec = {};

export {
  ...commonSpec,
  ...edgelessSpec
}
```

instead of disable eslint rules.